CHANGELOG-LATEST.md

```diff
@@ -1,5 +1,33 @@
-### Dependency Updates
-* [AUTOMATIC BUMP] Updates purchases-hybrid-common to 8.1.1 (#892) via RevenueCat Git Bot (@RCGitBot)
-  * [Android 7.2.7](https://github.com/RevenueCat/purchases-android/releases/tag/7.2.7)
-  * [iOS 4.31.2](https://github.com/RevenueCat/purchases-ios/releases/tag/4.31.2)
-* Bump danger from 9.4.0 to 9.4.1 (#889) via dependabot[bot] (@dependabot[bot])
+### New Features
+*   📱 Initial support for cross-platform RevenueCat Paywalls 🐾 🧱  (#852) 
+
+#### Instructions:
+- Update `purchases-flutter` in your `pubspec.yaml`:
+```yaml
+dependencies:
+  purchases_flutter: 6.6.0-beta.3
+```
+- For Android, you need to change your `MainActivity` to subclass `FlutterFragmentActivity` instead of `FlutterActivity`.
+
+#### Usage:
+```dart
+await Purchases.presentPaywallIfNeeded("pro");
+```
+
+#### Limitations:
+
+- Currently only full screen paywalls are supported
+- There is no way to detect paywall events other than using `addCustomerInfoUpdateListener`
+- Android's `minSdkVersion` is temporarily increased from `19` to `24` to support paywalls. This will be reverted in a future release as we split `purchases_flutter` and `purchases_flutter_ui`
+
+### Breaking changes from previous beta
+
+- `PurchasesFlutterActivity` has been removed. Use `FlutterFragmentActivity` provided by Flutter instead
```

Comment:
This is a breaking change from the previous beta... But I thought it would be better to remove it as soon as possible to avoid later confusions.